plugins/plugin-java/che-plugin-java-ext-debugger-java-client/src/main/java/org/eclipse/che/ide/ext/java/jdi/client/debug/remotedebug/RemoteDebugPresenter.java

```diff
@@ -38,6 +39,13 @@ public RemoteDebugPresenter(RemoteDebugView view, DebuggerPresenter debuggerPres
     /** Calls special method on view which shows dialog window. */
     public void showDialog() {
         view.show();
+        //view.setFocusInHostField(); // TODO fix behaviour. Because of animation/render we cannot set focus without delay
+        new Timer() {
```

Comment:
Most likely deferred command will not help in this case.
Need a delay > 300 so Timer is better to use there